packages/devtools_app/lib/src/config_specific/launch_url/launch_url.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'package:url_launcher/url_launcher.dart' as url_launcher;
+import '../../notifications.dart';
+import '_launch_url_stub.dart'
+    if (dart.library.html) '_launch_url_web.dart'
+    if (dart.library.io) '_launch_url_desktop.dart';
+
+Future<void> launchUrl(String url, BuildContext context) async {
+  if (isVSCodeEmbedded) {
+    launchUrlVSCode(url);
```

Comment:
can we call `launchUrlVSCode(url)` in addition to rather than instead of the regular url launch call?  My impression is that wouldn't be a problem for vscode as popups are blocked anyway. That way if we are wrong about an app actually being VSCode, it won't be a problem as the url will still launch if it can.

Reply:
Yep, done. Have tested in VSCode and in browser and this solution works.